365/client/js/router.js

```diff
@@ -36,15 +37,52 @@ Router.onBeforeAction(function() {
     }
 	}
 });
-
+*/
 
 Router.route("main",{
-    path:"/"
+  path:"/",
+  onBeforeAction: function() {
+    if (! Meteor.userId() ) {
+      Session.set('c_login', false);
+      this.redirect('splash');
+    } else {
+      this.next();
+    }
+  }
```

Comment:
Can you add in the check for Meteor.loggingIn()? We need it so that Meteor doesn't render templates before it defines the current user. Also I don't think you're using Session.set('c_login') so that can probably go.
Lastly a really nitpicky thing but can you make everything in this file either single quotes or double quotes? Whichever one doesn't really matter, as long as it's consistent